xrpl/docs/link-token.md

```diff
@@ -0,0 +1,179 @@
+```bash
+XRPL_PROVER_ADMIN=
+KEYRING_NAME=
+```
+## Deployment Steps
+
+### 1. Token Metadata Registration on XRPL Gateway
+```bash
+axelard tx wasm execute $XRPL_GATEWAY '{"register_token_metadata": {"xrpl_token": {"issued": {"currency": "'$XRPL_CURRENCY_CODE'", "issuer": "'$XRPL_ISSUER'"}}}}' -o text --from $XRPL_PROVER_ADMIN --keyring-backend $KEYRING_NAME  "${ARGS[@]}"
+```
+
+**Extract Values from Command Output:**
+```bash
+# Extract the following values from the previous transaction results
+MESSAGE_ID= #message_id
+PAYLOAD= #payload
+XRPL_TOKEN_ADDRESS='0x' +  #token_address
+
+**Extracted Values Example:**
+- **Message ID**: `0x8b49b5ccfb893269a5c263693805874cdeb3c932633ba0301094403c77dad839`
+- **Token Address**: `373336663663373634393533343933393030303030303030303030303030303030303030303030302e724e726a68314b475a6b326a42523377506641516e6f696474464659514b62516e32`
+- **Payload**: `00000000000000000000000000000000000000000000000000000000000000060000000000000000000000000000000000000000000000000000000000000060000000000000000000000000000000000000000000000000000000000000000f000000000000000000000000000000000000000000000000000000000000004b373336663663373634393533343933393030303030303030303030303030303030303030303030302e724e726a68314b475a6b326a42523377506641516e6f696474464659514b62516e32000000000000000000000000000000000000000000`
+```
+
+### 2. Execute Message on the Axelarnet Gateway
+**_NOTE:_**
+Manual relay operation can be executed by any funded axelar key including the xrpl multisig key
+```bash
+axelard tx wasm execute $AXELARNET_GATEWAY '{"execute": {"cc_id": {"source_chain": "xrpl", "message_id": "'$MESSAGE_ID'"}, "payload": "'$PAYLOAD'"}}' --from $XRPL_PROVER_ADMIN --keyring-backend $KEYRING_NAME "${ARGS[@]}"
+```
+
+### 3. Token Metadata Registration on Source Chain
+Managed EVM operation using the private key set in .env PRIVATE_KEY variable.
+```bash
+ts-node evm/its.js register-token-metadata $TOKEN_ADDRESS --gasValue $GAS_FEE
+```
+
+Wait for GMP Transaction to finish executing before proceeding
+
+### 4. Custom Token Registration on Source Chain
+**_NOTE:_**
+If no operator defined remove the --operator flag
+```bash
+ts-node evm/interchainTokenFactory.js --action registerCustomToken --tokenAddress $TOKEN_ADDRESS --tokenManagerType $TOKEN_MANAGER_TYPE --operator $OPERATOR --salt $SALT
+```
+Note: the GMP transaction is a two step process and only the first leg to the ITS Hub is required to succeed 
+
+From the output set the token Id for subsequent steps
+```bash
+TOKEN_ID= #tokenID from result without 0x prefix
+```
+
+### 5. Token Linking
+**_NOTE:_**
+If no operator defined set linkParams to empty bytes `0x`
+
+```bash
+ts-node evm/interchainTokenFactory.js --action linkToken --destinationChain xrpl --destinationTokenAddress $XRPL_TOKEN_ADDRESS --tokenManagerType $TOKEN_MANAGER_TYPE --linkParams $OPERATOR --salt $SALT --gasValue $GAS_FEE
```

Comment:
can you add just the solvBTC specific params/EVM cmds in a PR description based off of this that their team can run?